low-level/sensors/impl/generic/node_hw.py

```diff
@@ -260,7 +260,6 @@ def initialize(self, conf_reader, msgQlist, products):
             ServiceTypes.SERVER_NODE.value)
         self.__bmc_passwd = encryptor.decrypt(decryption_key,
             _bmc_secret, self.SENSOR_NAME)
-
```

Comment:
if no change, avoid this file from changed list

Reply:
removed.